### PR TITLE
Html skeleton - Sina Daryaei 

### PR DESCRIPTION
**Pull request by sinadaryaei**
image

### CODE REVIEW

File: html-skeleton/html-skeleton-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# html-skeleton-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Where do you put non-visible content about the HTML document?
+  By using the comment tags ```// comment```.
```
Comment (shawnkost) [inline html-skeleton/html-skeleton-quiz-notes.md:8]: The question is referring to which elements do you put non-visible content between? The `head` tag, the `body` tag, etc.

File: html-skeleton/html-skeleton-quiz-notes.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+- Where do you put visible content about the HTML document?
+  Within the ```<!DOCTYPE html>``` container.
```
Comment (shawnkost) [inline html-skeleton/html-skeleton-quiz-notes.md:11]: Not quite! Not all of these tags will be visible. Only certain ones. Check out the difference between `head` and `body`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a couple changes for your quiz notes!

File: html-skeleton/html-skeleton-quiz-notes.md (hunk 1 above)
Comment (sinadaryaei) [reply html-skeleton/html-skeleton-quiz-notes.md:8]: Using the ```<head>``` element and the ```<title>``` element.

File: html-skeleton/html-skeleton-quiz-notes.md (hunk 2 above)
Comment (sinadaryaei) [reply html-skeleton/html-skeleton-quiz-notes.md:11]: The visible content is placed in the ```<body>``` tag element.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**